modules/pricer/src/test/java/com/opengamma/strata/pricer/capfloor/VerticalSpreadVolatilityIborCapletFloorletBinaryPeriodPricerTest.java

```diff
@@ -0,0 +1,234 @@
+/*
+ * Copyright (C) 2022 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.pricer.capfloor;
+
+import static com.opengamma.strata.basics.currency.Currency.EUR;
+import static com.opengamma.strata.basics.index.IborIndices.EUR_EURIBOR_3M;
+import static com.opengamma.strata.collect.TestHelper.dateUtc;
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.LocalDate;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+import org.assertj.core.data.Offset;
+import org.junit.jupiter.api.Test;
+
+import com.opengamma.strata.basics.ReferenceData;
+import com.opengamma.strata.basics.currency.CurrencyAmount;
+import com.opengamma.strata.collect.timeseries.LocalDateDoubleTimeSeries;
+import com.opengamma.strata.collect.tuple.Pair;
+import com.opengamma.strata.market.param.CurrencyParameterSensitivities;
+import com.opengamma.strata.market.sensitivity.PointSensitivityBuilder;
+import com.opengamma.strata.pricer.rate.ImmutableRatesProvider;
+import com.opengamma.strata.product.capfloor.IborCapletFloorletBinaryPeriod;
+import com.opengamma.strata.product.capfloor.IborCapletFloorletPeriod;
+import com.opengamma.strata.product.rate.IborRateComputation;
+
+/**
+ * Test {@link VerticalSpreadVolatilityIborCapletFloorletBinaryPeriodPricer}.
+ */
+public class VerticalSpreadVolatilityIborCapletFloorletBinaryPeriodPricerTest {
```

Comment:
Test class and methods don't need to be public.

Reply:
I copy and paste old tests. I don't know what is the convention now. I remove the "public" markers from the abvoe tests.